app/locales/en.json

```diff
@@ -130,6 +131,9 @@
     "pauseUntilMorningInfo": "Specify till what hour 'Pause until morning' should pause.",
     "welcomeWindow": "Welcome window:",
     "showWelcomeWindow": "Show Welcome window on next start",
+    "opacityTheme": "Theme transparency:",
+    "opacityInfo": "If transparency is enabled in Preferences, how solid should the break window be ?",
```

Comment:
Only small change here: `How solid should the break window be?` According to native speakers, the `If ...` is not needed as it's pretty clear what we are talking about. 

Do you know how to squash commits to one? I think I can merge after :)

Reply:
Do you want me to also remove the `if ...` in the French translation ?

Reply:
Well I guess depends on the translator :) What do you think is more clear there...

Reply:
I prefer with the `if`, so I'm gonna leave it like that.